mm2src/coins/eth/eth_swap_v2/eth_taker_swap_v2.rs

```diff
@@ -18,6 +18,10 @@ use web3::types::{BlockNumber, TransactionId};
 const ETH_TAKER_PAYMENT: &str = "ethTakerPayment";
 const ERC20_TAKER_PAYMENT: &str = "erc20TakerPayment";
 const TAKER_PAYMENT_APPROVE: &str = "takerPaymentApprove";
+/// Maximum number of retry attempts for validating the token.
+const ATTEMPTS: usize = 3;
+/// Delay between call retries, in seconds.
+const RETRY_DELAY: f64 = 1.5;
```

Comment:
I think these names are pretty general and need to be more specific (or instead have these consts be local/close to where they are used) so as the doc comments.

Reply:
If they are used in a single place only (which is the case as far as I see), they can be placed inside the caller function directly.

Reply:
moved consts to the function scope and made their names a bit more specific 
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2334/commits/919f26daa28ed80cc7f10992477dcf69a9ff279c